src/Notifications/NotificationCenterController.cs

```diff
@@ -72,15 +78,29 @@ internal NotificationCenterController(DynamoView view, DynamoLogger dynLogger)
                 HorizontalOffset = notificationPopupHorizontalOffset,
                 VerticalOffset = notificationPopupVerticalOffset
             };
-            notificationUIPopup.webView.EnsureCoreWebView2Async();
-            notificationUIPopup.webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2InitializationCompleted;
             logger = dynLogger;
+            
             // If user turns on the feature, they will need to restart Dynamo to see the count
             // This ensures no network traffic when Notification center feature is turned off
             if (dynamoViewModel.PreferenceSettings.EnableNotificationCenter) 
             {
                 RequestNotifications();
-            }
+                InitializeBrowserAsync();
+            }   
+        }
+
+        async void InitializeBrowserAsync()
+        {
+            if (webBrowserUserDataFolder != null)
+            {
+                //This indicates in which location will be created the WebView2 cache folder
+                notificationUIPopup.webView.CreationProperties = new CoreWebView2CreationProperties()
+                {
+                    UserDataFolder = webBrowserUserDataFolder.FullName
+                };
+            }               
+            notificationUIPopup.webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2InitializationCompleted;
+            await notificationUIPopup.webView.EnsureCoreWebView2Async();
```

Comment:
This await should not be required right?

Reply:
@QilongTang well not sure if await is required or not but I knew that we should be awaiting async calls to finish otherwise we cannot control the flow of calls (if we need to use WebView2 later), for example in this case there is a call to RequestNotifications(); method before (in the original code was after), then not sure which problems could produce in the future but as you said let's test the behavior.
I will proceed to make the changes as you are suggesting and later we can do some experiments.
Thanks

Reply:
@RobertGlobant20 I made the change in https://github.com/DynamoDS/Dynamo/pull/13295 and running tests. Because I see this as one of the big difference before/after in your previous PR

Reply:
FYI: commit: https://github.com/DynamoDS/Dynamo/pull/13293/commits/9dcf6020b3606e3edca8444efa57cfb6d5176052

Reply:
@RobertGlobant20 For the `RequestNotifications()` call it should not affect WebView2 initialization, because the actual passing of information is bind to the event `NavigationCompleted`. So not until WebView2 was launched and navigated to the html, there would be no trial of passing info